TISServices/TISServices/Authorization/AuthorizeOpenAMAttribute.cs

```diff
@@ -56,7 +56,7 @@ public override void OnAuthorization(HttpActionContext actionContext)
              {
                  using (HttpClient client = new HttpClient())
                  {
-                     response = client.GetAsync(String.Format("{0}?access_token={1}", OpenAMUrl, authKey)).Result;
+                     response = client.GetAsync(String.Format("{0}?access_token={1}&realm=/sbac", OpenAMUrl, authKey)).Result;

```

Comment:
Correct, it didn't work without the realm. Should I take it out of this pull request?